doc/SAI_1.9.1 Release notes.md

```diff
@@ -0,0 +1,651 @@
+ */
+typedef sai_status_t (*sai_create_next_hop_group_map_fn)(
+        _Out_ sai_object_id_t *next_hop_group_map_id,
+        _In_ sai_object_id_t switch_id,
+        _In_ uint32_t attr_count,
+        _In_ const sai_attribute_t *attr_list);
+
+/**
+ * @brief Remove next hop group map
+ *
+ * @param[in] next_hop_group_map_id Next hop group map ID
+ *
+ * @return #SAI_STATUS_SUCCESS on success, failure status code on error
+ */
+typedef sai_status_t (*sai_remove_next_hop_group_map_fn)(
+        _In_ sai_object_id_t next_hop_group_map_id);
+
+/**
+ * @brief Set Next Hop Group map attribute
+ *
+ * @param[in] next_hop_group_map_id Next hop group map ID
+ * @param[in] attr Attribute
+ *
+ * @return #SAI_STATUS_SUCCESS on success, failure status code on error
+ */
+typedef sai_status_t (*sai_set_next_hop_group_map_attribute_fn)(
+        _In_ sai_object_id_t next_hop_group_map_id,
+        _In_ const sai_attribute_t *attr);
+
+/**
+ * @brief Get next hop group map attribute
+ *
+ * @param[in] next_hop_group_map_id Next hop group map ID
+ * @param[in] attr_count Number of attributes
+ * @param[inout] attr_list Array of attributes
+ *
+ * @return #SAI_STATUS_SUCCESS on success, failure status code on error
+ */
+typedef sai_status_t (*sai_get_next_hop_group_map_attribute_fn)(
+        _In_ sai_object_id_t next_hop_group_map_id,
+        _In_ uint32_t attr_count,
+        _Inout_ sai_attribute_t *attr_list);
+		
+	sai_create_next_hop_group_map_fn           create_next_hop_group_map;
+    sai_remove_next_hop_group_map_fn           remove_next_hop_group_map;
+    sai_set_next_hop_group_map_attribute_fn    set_next_hop_group_map_attribute;
+    sai_get_next_hop_group_map_attribute_fn    get_next_hop_group_map_attribute;
+	
+````````````````	
+
+````````````````	
+saiport.h
+
+
+ /**
+     * @brief Enable DSCP -> Forwarding Class MAP on port
+     *
+     * Map id = #SAI_NULL_OBJECT_ID to disable map on port.
+     * Default no map.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_QOS_MAP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_PORT_ATTR_QOS_DSCP_TO_FORWARDING_CLASS_MAP,
+
+    /**
+     * @brief Enable EXP -> Forwarding Class MAP on port
+     *
+     * Map id = #SAI_NULL_OBJECT_ID to disable map on port.
+     * Default no map.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_QOS_MAP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_PORT_ATTR_QOS_MPLS_EXP_TO_FORWARDING_CLASS_MAP,
+	
+````````````````	
+
+The PR related to this feature is available at PR#[1193](https://github.com/opencomputeproject/SAI/pull/1193)
+
+
+### Adding IPsec support
+
+A new API saiipsec.h has been added part of this current release to add an attribute for supporting both per SA and GLOBAL counters. Globals are aggregates for ingress and egress direction across all SAs. Per SA stats are directional per SA.
```

Comment:
This description is incorrect.  It should be replaced with something like:
A new IPsec API saiipsec.h has been added part of this current release.  For more detail, please refer to https://github.com/opencomputeproject/SAI/doc/IPsec/SAI_IPsec_API_proposal.md

Reply:
addressed

Reply:
@dipankar-ba  - please check, thx